modules/widgets/authors.php

```diff
@@ -172,22 +172,20 @@ public function widget( $args, $instance ) {
 				continue;
 			}
 
-			// Display a short list of recent posts for this author
-
+			// Display a short list of recent posts for this author.
 			if ( $r->have_posts() ) {
 				echo '<ul>';
 
 				while ( $r->have_posts() ) {
 					$r->the_post();
-					echo '<li><a href="' . get_permalink() . '">';
-
-					if ( get_the_title() ) {
-						echo get_the_title();
-					} else {
-						echo get_the_ID();
-					}
 
-					echo '</a></li>';
+					printf(
+						'<li><a href="%1$s" title="%2$s" %3$s>%4$s</a></li>',
```

Comment:
```suggestion
						'<li><a href="%1$s" title="%2$s"%3$s>%4$s</a></li>',
```

Avoids an extra space when it is not the page and the conditional output already adds one.